extensions/cli/debug/src/main/java/mil/nga/giat/geowave/cli/debug/CQLQuery.java

```diff
@@ -118,4 +119,11 @@ protected long runQuery(
 			return count;
 		}
 	}
+
+	@Override
+	public Object computeResults(
+			OperationParams params ) {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
problem (albeit merely debug commands, but still seems relatively straightforward to fix)